From: Matthias Clasen <mclasen@redhat.com>
Date: Sun, 26 Aug 2018 16:08:50 -0400
Subject: Fix up the flatpak-create-usb man page

There were some copy-paste leftovers here.

Closes: #2018
Approved by: alexlarsson

(cherry picked from commit a2069e75dfa312d569d15ffb46ab4dae631b919a)
---
 doc/flatpak-create-usb.xml | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/doc/flatpak-create-usb.xml b/doc/flatpak-create-usb.xml
index 59442da..9b5eded 100644
--- a/doc/flatpak-create-usb.xml
+++ b/doc/flatpak-create-usb.xml
@@ -104,7 +104,7 @@
                 <term><option>--user</option></term>
 
                 <listitem><para>
-                    Uninstalls from a per-user installation.
+                    Copy refs from the per-user installation.
                 </para></listitem>
             </varlistentry>
 
@@ -112,7 +112,7 @@
                 <term><option>--system</option></term>
 
                 <listitem><para>
-                    Uninstalls from the default system-wide installation.
+                    Copy refs from the default system-wide installation.
                 </para></listitem>
             </varlistentry>
 
@@ -120,7 +120,7 @@
                 <term><option>--installation=NAME</option></term>
 
                 <listitem><para>
-                    Copies refs from a system-wide installation specified by
+                    Copy refs from a system-wide installation specified by
                     <arg choice="plain">NAME</arg> among those defined in
                     <filename>/etc/flatpak/installations.d/</filename>.  Using
                     <arg choice="plain">--installation=default</arg> is
